_data/projects/athenaeum.yml

```
@@ -0,0 +1,8 @@
+name: Athenaeum 
+desc: A collection of various utility packages.
+site: https://aedart.github.io/athenaeum/
+tags:
+  - php
+upforgrabs:
+  name: good first issue
+  link: https://github.com/aedart/athenaeum/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```

:art: a simpler link format that will light up the integration a bit better:

```suggestion
  link: https://github.com/aedart/athenaeum/labels/good%20first%20issue
```